containers/dummy.rs

```diff
@@ -0,0 +1,5 @@
+pub fn main() {}
+
+/// Intentionally leave out pub to generate a warning so you know when this file is being used
+/// Read more here: https://blog.mgattozzi.dev/caching-rust-docker-builds/
+fn you_are_using_dummy_rs_for_docker_cache_grep_me() {}
```

Comment:
@Parth do we need the grep_me? unscalable pattern

Reply:
your call